scripts/build.py

```diff
@@ -19,17 +19,26 @@
 import shutil
 import subprocess
 import sys
-import yaml
-
+import hashlib
+import json
 
 HEAD_DIR = os.path.join('core', 'templates', 'dev', 'head', '')
 OUT_DIR = os.path.join('core', 'templates', 'prod', 'head', '')
 REMOVE_WS = re.compile(r'\s{2,}').sub
 YUICOMPRESSOR_DIR = os.path.join(
     '..', 'oppia_tools', 'yuicompressor-2.4.8', 'yuicompressor-2.4.8.jar')
 
-FILE_EXTENSIONS_TO_IGNORE = ['.py']
+FILE_EXTENSIONS_TO_IGNORE = ('.py')
```

Comment:
Done.

I initially thought the files that we can't rename would be spread through the folders, but as I look at it he only files that are in multiple folders are '.map' I will rework this to exclude from renaming  by folders/files locations, not file extensions.